OPAL/tac/src/main/scala/org/opalj/tac/fpcf/analyses/string/package.scala

```diff
@@ -0,0 +1,29 @@
+/* BSD 2-Clause License - see OPAL/LICENSE for details. */
+package org.opalj
+package tac
+package fpcf
+package analyses
+
+import org.opalj.br.DefinedMethod
+import org.opalj.br.Method
+import org.opalj.br.fpcf.properties.Context
+
+/**
+ * @author Maximilian Rüsch
+ */
+package object string {
+
+    type HighSoundness = Boolean
```

Comment:
Still not sure what the benefit of having this even as a type alias is. Why not just use boolean (note that, at least in the past, Scala tended to introduce lots of unnecessary boxing when using type aliases of primitive types).